main/wallet-api.md

```diff
@@ -0,0 +1,206 @@
+# Wallet API
+
+This document lists and describes the API commands available to the wallet.
+
+## Petnames
+
+The Agoric wallet uses *petnames* for issuers and purses.
+
+Petnames are your personal names for objects. No one else can see or
+modify a petname without your permission. You can think of them as
+your phone's contacts list. The actual phone number is what your phone
+uses to call someone, but for you to more easily tell who a number is
+associated with, you've assigned a petname to it, such as Mom,
+Grandpa, Kate S., etc.
+
+## Overview
+
+Wallet API commands work with the following object types:
+- `purse`: Stores assets until you withdraw them into a payment for use 
+- `issuer`: Creates empty purses and payments and maps minted assets
+  to them when assets are added or removed. Issuers verify and move
+  digital assets.
+- `offer`: Consists of what amount of what brand you're willing to
+  give, and what amount of what brand you want, as well as the
+  conditions under which the offer holder can cancel it.
+
+## Purse API Methods
+
+### `deposit()`
+- **Function:**
+  - Deposits a payment into the specified purse in the wallet. Returns the deposited amount.
+- **Required Arguments:**
+  - `pursePetname`
+    - **String**. The petname of the purse in which to deposit the specified payment.
+  - `payment`
+    - **Object**. A payment object, which is deposited into the specified purse.
+- **Optional Arguments:**
+  - None.
+
+### `getPurses()`
+- **Function:**
+  - Returns all purse objects and their petnames in the wallet. They are returned as an array of [petname, purseObject] arrays which can be turned into a `Map` (`new Map(getPurses())`)
+- **Required Arguments:**
+  - None.
+- **Optional Arguments:**
+  - None.
+
+### `getPurse(pursePetname)`
+- **Function:**
+  - Returns the `purse` object with the given petname
```

Comment:
Rather than "with", it's really that one assigned to that pet name.  It's more that the petname has a purse, rather than the purse having a petname. I'm a little unsure about how we want this usage, so if wallet comments speak to it, use that slant.

Reply:
I think this usage makes sense, since it will always be a bidirectional one-to-one mapping.